src/components/settings/Calendars.js

```diff
@@ -112,7 +130,8 @@ export default class Calendars extends AddDeleteSetting {
         newItem = {
           uid: uuid(),
           type,
-          name: valueOfAdd,
+          name: this.state.calendarName || valueOfAdd,
+          hexColor: this.state.hexColor || null,
```

Comment:
This does not make sense, if hexColor is null than we can use hexColor

Reply:
Yes, i removed that part.